config/routes.rb

```diff
@@ -5,7 +5,7 @@
 
   get 'my-modules', to: 'learning#show' # @see User#course
   get 'about-training', to: 'training/modules#index', as: :course_overview
-  get 'gov-one/info', to: 'gov_one#info'
+  get 'gov-one/info', to: 'gov_one#info', as: :gov_one_info
```

Comment:
Out of curiosity what name was it given before?

Reply:
Good point! Not necessary